chapter02/qli/ex5.pl

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/perl
+
+use warnings;
+
+my $str=<STDIN>;
+chomp($str);
+
+my $num=<STDIN>;
+chomp($num);
+
+print "Ex Result:: \n";
+if ( $num && $num>0 ){
```

Comment:
@MarkWignall I don't think input validation is part of the requirements, I didn't add validation to all exercises too. I should have added more detailed program prompts.